dali/pipeline/operator/op_schema.cc

```diff
@@ -78,6 +78,37 @@ to accommodate a batch of samples of this size.)code",
   AddOptionalArg("preserve", R"code(Prevents the operator from being removed from the
 graph even if its outputs are not used.)code",
                  false);
+
+
+  // For simplicity we pass StackSummary as 4 separate arguments so we don't need to extend DALI
+  // with support for special FrameSummary type.
+  // List of FrameSummaries can be reconstructed using utility functions.
+  AddOptionalArg("_origin_stack_filename", R"code(Every operator defined in Python captures and
+processes the StackSummary (a List[FrameSummary], defined in Python traceback module) that describes
```

Comment:
...I will complain about the use of the name "summary" here like I did at the point of declaration of the FrameSummary structure.